jaxrs/src/main/java/org/jboss/as/jaxrs/JaxrsConstants.java

```diff
@@ -24,10 +24,16 @@ public class JaxrsConstants {
     public static final String RESTEASY_GZIP_MAX_INPUT = "resteasy-gzip-max-input";
     public static final String RESTEASY_JNDI_RESOURCES = "resteasy-jndi-resources";
     public static final String RESTEASY_LANGUAGE_MAPPINGS = "resteasy-language-mappings";
+    public static final String RESTEASY_MATCH_CACHE_ENABLED = "resteasy-match-cache-enabled";
```

Comment:
Do we want to use the `resteasy-` prefix on these? I don't have a strong opinion, but leaving it off is less typing for administrators. I left it off for the tracing attributes.

Reply:
I can't remember where, but somewhere there's a transformation from, e.g., "resteasy-language-mappings" to "resteasy.language.mappings". Wouldn't omitting "resteasy" screw that up?

Reply:
That's easily fixable though :) This is how I did it for the tracing attributes. I just simply added them as context parameters https://github.com/wildfly/wildfly/blob/1f68b0ad54b7899650a782a8efdde0cec196865c/jaxrs/src/main/java/org/jboss/as/jaxrs/JaxrsSubsystemAdd.java#L138-L143.

Reply:
Oh, I see. Well, I have mixed feelings. Once could argue that the tracing parameters are a class unto themselves, but the six new parameters are more like the existing ones, which introduces two different conventions.

Reply:
Yeah. It's more opinion UX than real UX probably :)

Reply:
What if we add a shortened version of **every** parameter, all at one time, in a new feature request. Then the old names work and each has a shortened version as well.

Reply:
We can just keep the `resteasy` prefix. There isn't really a way to allow for two different attribute names that I'm aware of.